src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions_CreateEqualsMethod.cs

```diff
@@ -375,8 +375,8 @@ private static ITypeSymbol GetType(Compilation compilation, ISymbol symbol)
         {
             switch (symbol)
```

Comment:
i feel liek i've seen this helper before :)

Reply:
Yeah, you clearly liked writing it over and over again. 😛

Reply:
(this particular one has a "return object" which I'm not sure exactly how that gets hit...)